CMake configuration error when building against VTK with external HDF5

When building against VTK with external HDF5:

- `VTK_MODULE_ENABLE_VTK_hdf5:STRING=YES`
- `VTK_MODULE_USE_EXTERNAL_VTK_hdf5:BOOL=ON`

the following CMake error is raised:

```
CMake Error at /home/aure/src/vtk/build/lib64/cmake/vtk-9.0/patches/99/FindHDF5.cmake:222 (try_compile):
  Unknown extension ".c" for file

    /home/aure/src/vtkAddon/build/CMakeFiles/hdf5/cmake_hdf5_test.c

  try_compile() works only for enabled languages.  Currently these are:

    CXX

  See project() command to enable other languages.
Call Stack (most recent call first):
  /home/aure/src/vtk/build/lib64/cmake/vtk-9.0/patches/99/FindHDF5.cmake:556 (_HDF5_test_regular_compiler_C)
  /home/aure/src/vtk/build/lib64/cmake/vtk-9.0/VTK-vtk-module-find-packages.cmake:303 (find_package)
  /home/aure/src/vtk/build/lib64/cmake/vtk-9.0/vtk-config.cmake:138 (include)
  /home/aure/src/vtk/build/vtk-config.cmake:1 (include)
  CMakeLists.txt:24 (find_package)
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -3,7 +3,7 @@ cmake_minimum_required(VERSION 3.13.4)
 #-----------------------------------------------------------------------------
 
 project(vtkAddon
-  LANGUAGES CXX
+  LANGUAGES CXX C
   DESCRIPTION "General-purpose features that may be integrated into VTK library in the future."
 )
 

